lib/diagnostic.rb

```diff
@@ -93,13 +95,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.slice(-2, arr.length)'
 
```

Comment:
This will return the last 2 elements in the array, but it would not remove them
You could remove them by saying `arr.pop 2`